### PR TITLE
Add new player core preview spells, alter existing spells

### PR DESCRIPTION
**Pull request by TikaelSol**
Before merging we need to make sure all swapped spells have their ID properly swapped to the new spell

### CODE REVIEW

File: packs/spells/blazing-bolt.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
     "_id": "ZxHC7V7HtjUsB8zH",
+    "folder": "sspn0dATguOE2tTz",
```
Comment (stwlam) [inline packs/spells/blazing-bolt.json:3]: ?

Comment (TikaelSol) [reply packs/spells/blazing-bolt.json:3]: Ugh, Foundry cruft.

File: packs/spells/oaken-resilience.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
-{
+{
```
Comment (TikaelSol) [inline packs/spells/oaken-resilience.json:1]:
```suggestion
{
```
No idea

File: packs/spells/thunderstrike.json [hunk 3]
```diff
@@ -1,11 +1,8 @@
 {
-    "_id": "jrBa9deU2ULFWvSl",
-    "img": "systems/pf2e/icons/spells/meteor-swarm.webp",
-    "name": "Meteor Swarm",
+    "_id": "zDJS8E66UI0himqV",
+    "img": "systems/pf2e/icons/spells/shocking-grasp.webp",
+    "name": "Thunderstrike",
```
Comment (TikaelSol) [inline packs/spells/thunderstrike.json:4]: wut

**Pull request merged**